### PR TITLE
[onnx]maxpool1d

### PR DESCRIPTION
**Pull request by xiaxin1998**

### CODE REVIEW

**Comment by xiaxin1998** [comment]
/view/ZOO-PR/job/ZOO-PR-Validation/1164/ Jenkins passed .  @zhichao-li

File: pyzoo/test/zoo/pipeline/onnx/test_model_loading.py [hunk 1]
```diff
@@ -591,3 +595,45 @@ def test_concat(self):
...
+        output = OnnxLoader.run_node(node, [x])
+        np.testing.assert_almost_equal(output["y"], y, decimal=5)
+
+    def test_maxpool1d_strides(self):
+        node = onnx.helper.make_node(
+            'MaxPool',
+            inputs=['x'],
+            outputs=['y'],
+            kernel_shape=[2],
+            strides=[2]
```
Comment (zhichao-li) [inline pyzoo/test/zoo/pipeline/onnx/test_model_loading.py]: is there any unitest to conver the case that `pads` is not None?

File: pyzoo/test/zoo/pipeline/onnx/test_model_loading.py [hunk 2]
```diff
@@ -591,3 +595,45 @@ def test_concat(self):
...
+        strides = [2]
+        out_shape = pool_op_common.get_output_shape('VALID', x_shape[2:], kernel_shape, strides)
+        padded = x
+        y = pool_op_common.pool(padded, x_shape, kernel_shape, strides, out_shape, [0], 'MAX')
+        output = OnnxLoader.run_node(node, [x])
+        np.testing.assert_almost_equal(output["y"], y, decimal=5)
+
+    def test_onnx_maxpool1d(self):
+        pytorch_model = torch.nn.Sequential(
+            torch.nn.MaxPool1d(2)
```
Comment (zhichao-li) [inline pyzoo/test/zoo/pipeline/onnx/test_model_loading.py]: specify `border_mode` here to cover the path of using `boarder_mode` ?

File: pyzoo/zoo/pipeline/api/onnx/onnx_helper.py [hunk 3]
```diff
@@ -71,9 +71,12 @@ def get_padds(onnx_attr):
 
         if "pads" in onnx_attr.keys():
             pads4 = [int(i) for i in onnx_attr["pads"]]
```
Comment (zhichao-li) [inline pyzoo/zoo/pipeline/api/onnx/onnx_helper.py]: rename it to `pads`

File: pyzoo/zoo/pipeline/api/onnx/mapper/maxpool.py [hunk 4]
```diff
@@ -41,5 +41,20 @@ def _to_tensor(self):
                                            border_mode=border_mode,
                                            pads=pads)
             return maxpool(self.model_inputs[0].zvalue)
+        elif (rank == 3):
```
Comment (zhichao-li) [inline pyzoo/zoo/pipeline/api/onnx/mapper/maxpool.py]: add checking for `storage_order`

**Comment by xiaxin1998** [comment]
18888/view/ZOO-PR/job/ZOO-PR-Validation/1196/  Jenkins passed. @zhichao-li

File: zoo/src/main/scala/com/intel/analytics/zoo/pipeline/api/keras/layers/MaxPooling1D.scala [hunk 5]
```diff
@@ -46,13 +47,21 @@ class MaxPooling1D[T: ClassTag](
     override val poolLength: Int = 2,
     override val stride: Int = -1,
     override val borderMode: String = "valid",
-    override val inputShape: Shape = null)(implicit ev: TensorNumeric[T])
+    override val inputShape: Shape = null,
+    val pads: Int = 0)(implicit ev: TensorNumeric[T])
```
Comment (zhichao-li) [inline zoo/src/main/scala/com/intel/analytics/zoo/pipeline/api/keras/layers/MaxPooling1D.scala]: pad?

File: zoo/src/main/scala/com/intel/analytics/zoo/pipeline/api/keras/python/PythonZooKeras.scala [hunk 6]
```diff
@@ -572,8 +572,11 @@ class PythonZooKeras[T: ClassTag](implicit ev: TensorNumeric[T]) extends PythonB
       poolLength: Int = 2,
       stride: Int = -1,
       borderMode: String = "valid",
-      inputShape: JList[Int] = null): MaxPooling1D[T] = {
-    MaxPooling1D(poolLength, stride, borderMode, toScalaShape(inputShape))
+      inputShape: JList[Int] = null,
+      pads: Int = 0): MaxPooling1D[T] = {
```
Comment (zhichao-li) [inline zoo/src/main/scala/com/intel/analytics/zoo/pipeline/api/keras/python/PythonZooKeras.scala]: pad?

**Comment by xiaxin1998** [comment]
@zhichao-li  18888/view/ZOO-PR/job/ZOO-PR-Validation/1261/ Jenkins Passed

**Pull request merged**